Makefile

```diff
@@ -89,10 +89,15 @@ test-unit: ./vendor ## run unit tests
 	@go test -failfast -v -cover ./...
 
 .PHONY: update-golden
-update-golden: ./vendor ## run unit tests (updating golden files)
+update-golden: clean-golden ./vendor ## run unit tests (updating golden files)
 	@echo "Running unit tests to update golden files..."
 	@./hack/update-golden.sh
 
+.PHONY: clean-golden
+clean-golden:
+	@echo "Clean golden files..."
+	-find -iname "*.golden" -delete
```

Comment:
Just one nit -> exclude vendor, right now if don't have any file but may be in furture

Reply:
good point